zds/tutorialv2/views/views_published.py

```diff
@@ -527,7 +528,7 @@ def post(self, request, *args, **kwargs):
             reaction.hide_comment_by_user(self.request.user, text)
             return redirect(reaction.get_absolute_url())
         except (IndexError, ValueError, MultiValueDictKeyError):
-            raise Http404
+            raise Http404("")
```

Comment:
"You cannot hide this" ;)

(ce `except` est particulièrement mal foutu, mais c'est ma faute)